src/main.ts

```diff
@@ -42,12 +42,13 @@ async function run(): Promise<void> {
       branchName = process.env.GITHUB_REF?.split('/')
         .slice(2)
         .join('/')
-        .replace(/\//g, '-')
+        .replace(/[^0-9a-z]/g, '-')
```

Comment:
What if the branch contains upper case characters?

Reply:
should be ok?

Reply:
As in it will not prevent it from working

Reply:
In fact, Alex gave us an example already 🤣